src/index.html

```diff
@@ -11,7 +11,112 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form 
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
+      method="post"
+      autocomplete="off"
+    >
+      <fieldset class="tabel">
+        <legend>Personal information</legend>
+        <label class="form__field">
+          Surname:
+          <input type="text" name="Surname">
+        </label>
+        <label class="form__field">
+          Name:
+          <input type="text" name="Name">
+        </label>
+        <label class="form__field">
```

Comment:
```suggestion
      <fieldset class="tabel">
        <legend>Personal information</legend>
        
        <label class="form__field">
          Surname:
          <input type="text" name="Surname">
        </label>
        
        <label class="form__field">
          Name:
          <input type="text" name="Name">
        </label>
        
        <label class="form__field">
```
Add newlines between adjacent blocks of code to make your code easier to read

Reply:
Здається все вийшло, тести пройшли)